### ISSUE TITLE
عدم قطع اتصال در نسخه اندروید برای اتصال به یک ایپی دیگر

### ISSUE THREAD

**Issue by MehrdadfreeIran** [bepass-org/oblivion#163]
برای اتصال مجدد به یک ایپی دیگر اتصال اولیه قطع نمیشود
To reconnect to another IP, the initial connection is not interrupted

**Comment by markpash** [comment]
Can you please provide more details?
Can you please provide some logs?

**Comment by MehrdadfreeIran** [comment]
Yes, when I click on the disconnect button, the connection does not disconnect, so I create a new connection to increase the speed

**Comment by MehrdadfreeIran** [comment]
With a long connection to one IP, the speed decreases, to increase the speed, you have to go to another IP But the initial connection is not disconnected (the software has a bug here) until a new connection is created
